Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -44,6 +44,8 @@ public class PlayerMovement : MonoBehaviour
     [SerializeField]
     private float jumpTimeout = 0.5f;
 
+    private const float marsGravity = 3.72076f;
```

Comment:
Not sure if realism in gravity constant is what we want to prioritize

Reply:
Agreed, we might tweak this value later when we move to more vertical level design I think, but this feels OK for now I think